public/app/features/dashboard/api/dashboard_api.ts

```diff
@@ -0,0 +1,84 @@
+import { config, getBackendSrv } from '@grafana/runtime';
+import { ScopedResourceClient } from 'app/features/apiserver/client';
+import { ResourceClient } from 'app/features/apiserver/types';
+import { SaveDashboardCommand } from 'app/features/dashboard/components/SaveDashboard/types';
+import { dashboardWatcher } from 'app/features/live/dashboard/dashboardWatcher';
+import { DeleteDashboardResponse } from 'app/features/manage-dashboards/types';
+import { DashboardDTO, DashboardDataDTO } from 'app/types';
+
+export interface DashboardAPI {
+  /** Get a dashboard with the access control metadata */
+  getDashboardDTO(uid: string): Promise<DashboardDTO>;
```

Comment:
> is there going to be a getDashboard()...

Ideally yes -- and it would be the raw resource without munging its properties to match the existing DTO shape